core/src/main/scala/tech/beshu/ror/accesscontrol/domain.scala

```diff
@@ -461,28 +463,47 @@ object domain {
       Action.isInternal(value)
   }
   object Action {
-    val searchAction = Action("indices:data/read/search")
-    val mSearchAction = Action("indices:data/read/msearch")
-    val restoreSnapshotAction = Action("cluster:admin/snapshot/restore")
-    val fieldCapsAction = Action("indices:data/read/field_caps")
-    val asyncSearchAction = Action("indices:data/read/async_search/submit")
-    val rollupSearchAction = Action("indices:data/read/xpack/rollup/search")
-    val searchTemplateAction = Action("indices:data/read/search/template")
-    val putTemplateAction = Action("indices:admin/template/put")
-    val putIndexTemplateAction = Action("indices:admin/index_template/put")
-    val getSettingsAction = Action("indices:monitor/settings/get")
-    val monitorStateAction = Action("cluster:monitor/state")
     // ROR actions
-    val rorUserMetadataAction = Action("cluster:ror/user_metadata/get")
-    val rorConfigAction = Action("cluster:ror/config/manage")
-    val rorTestConfigAction = Action("cluster:ror/testconfig/manage")
-    val rorAuthMockAction = Action("cluster:ror/authmock/manage")
-    val rorAuditEventAction = Action("cluster:ror/audit_event/put")
-    val rorOldConfigAction = Action("cluster:ror/config/refreshsettings")
+    val rorUserMetadataAction: Action = rorAction("user_metadata/get")
+    val rorConfigAction: Action = rorAction("config/manage")
+    val rorTestConfigAction: Action = rorAction("testconfig/manage")
+    val rorAuthMockAction: Action = rorAction("authmock/manage")
+    val rorAuditEventAction: Action = rorAction("audit_event/put")
+    val rorOldConfigAction: Action = rorAction("config/refreshsettings")
```

Comment:
in the sealed trait hierarchy, this will be the same that the `rorConfigAction` is.

Reply:
We need to maintain both versions because we have two actions in elasticsearch modules: `tech.beshu.ror.es.actions.rrconfig.RRConfigActionType` `tech.beshu.ror.es.actions.rradmin.RRAdminActionType`